modules/siwe.ts

```diff
@@ -48,68 +50,119 @@ class Siwe {
       _id: walletAddress
     };
     const newEthAccountXhr = await privateBaseAxios.post('/account/ethereum/create', data);
-    console.log({ newEthAccountXhr })
     const newEthAccount = newEthAccountXhr.data
 
     /* Update user_id of newEthAccount */
     const updateData = {
       user_id: userId
     }
-    console.log({ updateData })
     const updatedAccountXhr = await privateBaseAxios.patch(`/account/ethereum/update/${newEthAccount._id}`, updateData)
     return updatedAccountXhr.data
   }
 
-  static async signInWithEthereum(discordId: string) {
+  static encodeAddress (address: string): string {
```

Comment:
Can you add some comments above each of these methods just for quick reference on what they do. Few similar sounding methods in this file